physionet-django/console/views.py

```diff
@@ -1155,6 +1155,18 @@ def process_credential_application(request, application_slug):
 
     ref_email = notification.contact_reference(request, application,
                                                send=False,  wordwrap=False)
+    all_active_users = User.objects.filter(is_active=True)
```

Comment:
Instead of iterating emails, you could query for the user object directly. e.g. 

```
from django.core.exceptions import ObjectDoesNotExist

try:
    ref = User.objects.get(associated_emails__email=email.lower(),
                           associated_emails__is_verified=True)
    known_active = True
    known_cred = ref.is_credentialed
except ObjectDoesNotExist:
    known_active = False
    known_cred = False
```

Reply:
Much cleaner thanks!